http-clients/aws-crt-client/src/test/java/software/amazon/awssdk/http/crt/AwsCrtHttpClientSpiVerificationTest.java

```diff
@@ -132,6 +133,27 @@ public void requestFailed_connectionTimeout_shouldWrapException() {
         }
     }
 
+    @Test
+    public void requestFailed_notRetryable_shouldNotWrapException() {
```

Comment:
Looks like this test failed on CodeBuild

```
java.lang.NullPointerException: RequestContentPublisher must not be null.
--
at software.amazon.awssdk.http.crt.AwsCrtHttpClientSpiVerificationTest.requestFailed_notRetryable_shouldNotWrapException(AwsCrtHttpClientSpiVerificationTest.java:151)

```

Reply:
Fixed with a mock class.